src/js/src/components/Toolbox.vue

```diff
@@ -7,22 +7,22 @@
       <div class="toolboxNavigation">
         <h3>
           <span class="toolboxIcon" />
-          Available tools:
+          {{ $t('toolBox:available_tools') }} :
         </h3>
         <button :class="currentTool === 'wordcloud' ? 'activeTool' : ''" @click="setCurrentTool('wordcloud')">
-          Wordcloud
+          {{ $t('toolBox:wordcloud') }} 
         </button>
         <button :class="currentTool === 'linkgraph' ? 'activeTool' : ''" @click="setCurrentTool('linkgraph')">
-          Link graph
+          {{ $t('toolBox:link_graph') }} 
         </button>
         <button :class="currentTool === 'domainstats' ? 'activeTool' : ''" @click="setCurrentTool('domainstats')">
-          Domain stats
+          {{ $t('toolBox:domain_stats') }}
         </button>
         <button :class="currentTool === 'gephiexport' ? 'activeTool' : ''" @click="setCurrentTool('gephiexport')">
-          Link graph Gephi export
+          {{ $t('toolBox:link_graph_gephi_export') }}
         </button>
         <button :class="currentTool === 'ngramnetarchive' ? 'activeTool' : ''" @click="setCurrentTool('ngramnetarchive')">
-          Ngram Netarchive 
+          {{ $t('toolBox:ngram_netarchive') }}
         </button>
```

Comment:
Vuei18n supports that the locale message resource key for the translation function can be specified for a specific resource namespace with using '.' (dot) notation just like a JavaScript object. So the above resources can be structured as:

`"toolbox": {
    "available_tools": "Dostępne narzędzia",
    "wordcloud": "Chmura słów"
  "link_graph":"Sieć linków"
....
  },`

and referenced with `toolbox.available_tools` in the code.